draft-ietf-rats-corim.md

```diff
@@ -1491,36 +1491,97 @@ and how the verifier matches a Reference Value against the Accepted Claims Set
 
 #### Grouping Reference Values and Endorsements {#sec-grouping-ref-vals}
 
-> This paragraph will be replaced by a description of how the CoRIM tells the
-verifier which Reference Values and Endorsed Values are grouped together.
+> *TODO: This paragraph will be replaced by a description of how the CoRIM tells the
+verifier which Reference Values and Endorsed Values are grouped together.*
 
 [^issue]: Need to get agreement on how group membership is encoded. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/136
 
-[^issue]: Need to describe how to match conditional endorsements. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/80
+> *TODO: Can we find a better term than the placeholder "group"? If so then I think we should replace all mentions of "group" with the new term.*
 
 #### Matching all Reference Values in a group against the Accepted Claims Set {#sec-match-all-ref-vals}
 
-If all Reference Values in a group match entries in the Accepted Claims Set
-then all Endorsements in the group are added to the Accepted Claims Set
-(see {{sec-add-to-acs}}). {{sec-match-one-ref-val}} describes how one
+If a group contains at least one `reference-triple-record` and at least one
+`endorsed-triple-record` then the Reference Values contained in the former
+are used to decide whether to add Endorsed Values in the latter to the
+Accepted Claims Set.
+
+The Verifier SHALL try to match all Reference Values contained within all
+`reference-triple-record`s within the group against Evidence and Endorsements
+in the Accepted Claims Set. {{sec-match-one-ref-val}} describes how each
 Reference Value is matched against the Accepted Claims Set.
 
+If all Reference Values in a group match entries in the Accepted Claims Set
+then all Endorsed Values from `endorsed-triple-record`s in the group are added
+to the Accepted Claims Set (see {{sec-add-to-acs}}).
+
 If any Reference Value in a group does not match the Accepted Claims Set then
```

Comment:
I think "Accepted Claims Set"  is ambiguous given the above statement that ACS may contain Endorsements as well as Evidence. The algorithm for accepting Evidence applies to Reference Values. If the Attester supplies no Evidence that is vetted by Reference Values, then no Endorsed values should be attributed IMO. Simple example, if Attester asserts it is an HSM, but no Ref Vals substantiate this, and a FIPS certifying body produces an Endorsement that says HSM is EAL4+, then the Attester has successfully achieved privilege escalation.

Reply:
We should probably discuss this more. My take is that Evidence is automatically added to the ACS with the authorization of the attester. Or more precisely with the authority key which authorized the attester (eg Intel DICE root if the verifier trusts that key).
For most relying parties, that evidence means nothing without a CoRIM to add a conditional endorsement. The FIPS certifying body could say that an entity with measurements A=A1 and B=B1 authorized by key K is FIPS certified.
As we discussed, I think authorized by deserves its own section.

Reply:
I will address this as part of the patch for issue #140